src/py-opentimelineio/opentimelineio/adapters/cmx_3600.py

```diff
@@ -665,14 +673,28 @@ def _expand_transitions(timeline):
                 transition_duration.rate
             )
 
+            # Because of how comments fall on transition events, some of the
```

Comment:
Can this be more specific? "Because comments in an EDL are listed after transition events" 
(rather than "how comments fall")